contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,876 @@
+        }
+
+        {
+            address FLUID_ETH_DEX = getDexAddress(10);
+            {
+                // cbBTC-ETH DEX
+                {
+                    setSupplyProtocolLimitsPaused(FLUID_ETH_DEX, FLUID_ADDRESS);
+
+                    setSupplyProtocolLimitsPaused(FLUID_ETH_DEX, ETH_ADDRESS);
+                }
+            }
+
+            {
+                // Pause user supply and borrow
+                address[] memory supplyTokens = new address[](2);
+                supplyTokens[0] = FLUID_ADDRESS;
+                supplyTokens[1] = ETH_ADDRESS;
+
+                address[] memory borrowTokens = new address[](0);
+
+                // Pause the user operations
+                LIQUIDITY.pauseUser(FLUID_ETH_DEX, supplyTokens, borrowTokens);
+            }
+        }
+    }
+
+    // @notice Action 8: Set Launch Limits for RLP-USDC DEX
+    function action8() internal isActionSkippable(8) {
+        address RLP_USDC_DEX = getDexAddress(28);
+        {
+            // RLP-USDC DEX
+            DexConfig memory DEX_RLP_USDC = DexConfig({
+                dex: RLP_USDC_DEX,
+                tokenA: RLP_ADDRESS,
+                tokenB: USDC_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 6_500_000, // $6.5M
+                baseBorrowLimitInUSD: 0, // $0
+                maxBorrowLimitInUSD: 0 // $0
+            });
+            setDexLimits(DEX_RLP_USDC); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(RLP_USDC_DEX, TEAM_MULTISIG, false);
+        }
+
+        {
+            address fSL28_RLP_USDC = getSmartLendingAddress(28);
+
+            // set rebalancer at fSL28 to reserve contract proxy
+            ISmartLendingAdmin(fSL28_RLP_USDC).setRebalancer(
+                address(FLUID_RESERVE)
+            );
+        }
+        {
+            IFluidDex(RLP_USDC_DEX).updateMaxSupplyShares(
+                5_000_000 * 1e18 // $5M
+            );
+        }
+    }
+
+    // @notice Action 9: Set Launch Limits for Gold Vaults and DEX
+    function action9() internal isActionSkippable(9) {
+        {
+            // PAXG-XAUT DEX
+            address PAXG_XAUT_DEX = getDexAddress(32);
+            {
+                DexConfig memory DEX_PAXG_XAUT = DexConfig({
+                    dex: PAXG_XAUT_DEX,
+                    tokenA: PAXG_ADDRESS,
+                    tokenB: XAUT_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 1_000_000, // $1M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_PAXG_XAUT); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(PAXG_XAUT_DEX, TEAM_MULTISIG, false);
+            }
+            {
+                IFluidDex(PAXG_XAUT_DEX).updateMaxSupplyShares(
+                    1_800_000 * 1e18 // $1.8M
```

Comment:
this will have to be adjusted 

1 share should be 2 Gold units, so ~6500$ , so around 170 shares needed

Reply:
i think it will be 276.9 or ~277 shares